### ISSUE TITLE
The working example for ARIA9 - Example 3 has problems

### ISSUE THREAD

**Issue by dd8** [w3c/wcag#1148]
https://www.w3.org/WAI/WCAG21/working-examples/aria-labelledby-workshop-booking-timetable/

There are 2 tables in this working example:

- the dinosaur conference table from ARIA9
- a second layout table containing the text field labelled 'Maiden name' (which isn't part of example 3 in ARIA9)

Smart quotes (U+201D) have been used for the role attribute value on the second table instead of ASCII U+0022 double-quote. This triggers the following message in the W3 validator:

> an attribute value must be a literal unless it contains only name characters

This is parsed as `role="”presentation”"` in the DOM (with the smart quotes as part of the role name) so the role is unrecognised and falls back to the native table role in Firefox/Chrome/Edge. In Safari layout table heuristics kick-in which stops it being exposed as a table.

Might be simplest to just remove the second table since it doesn't add anything to the example (and some people may be unhappy with the field label).

**Comment by dd8** [comment]
There's also an argument that the 'Log in' button fails 2.4.6: Headings and Labels, since the user is probably expecting to Book or Register

**Comment by patrickhlauke** [comment]
> There's also an argument that the 'Log in' button fails 2.4.6: Headings and Labels, since the user is probably expecting to Book or Register

it wouldn't fail if pressing that button actually does go to an intermediate "Log in" step first, so not something that can be determined in isolation. But yes for clarity/to make the example actually make more real-world sense, changing it to "Register" would avoid that kind of conundrum.

**Comment by detlevhfischer** [comment]
Proposed working group response:
Thank you for your comment.
We have updated the example code by replacing smart quotes with standard quotes.
https://github.com/w3c/wcag/pull/1165
